packages/cli-exec/src/exec.js

```diff
@@ -59,8 +59,12 @@ export const exec = command('exec', {
     log.warn('Percy is disabled');
   } else {
     try {
-      percy.projectType = percy.client.tokenType();
-      percy.skipDiscovery = percy.shouldSkipAssetDiscovery(percy.projectType);
+      // Skip this for app because they are triggered as app:exec
+      // Remove this once they move to exec command as well
+      if (percy.projectType !== 'app') {
```

Comment:
This will always be true right?

projectType is defined (check above)

Can you explain when will it be false?